client/state/stats/lists/constants.js

```diff
@@ -0,0 +1,26 @@
+export const PUBLICIZE_SERVICES_LABEL_ICON = {
+	twitter: {
+		label: 'Twitter',
```

Comment:
Maybe overkill, especially if these services are hard-coded anyways, but someday might be nice to incorporate the [`/meta/external-services` endpoint](https://developer.wordpress.com/docs/api/1.1/get/meta/external-services/) into Redux state to grab these labels.